src/core/aws-net-websocket-registry.adb

```diff
@@ -286,8 +287,16 @@ package body AWS.Net.WebSocket.Registry is
                      DB.Get_Socket (WS_Id, WS);
 
                      if WS /= null then
-                        DB.Remove (WS);
-                        Message_Queue.Add (WS_Id);
+                        --  We should never fail to ensure the WebSocket is
+                        --  properly released.
+                        begin
+                           DB.Remove (WS);
+                           Message_Queue.Add (WS_Id);
+                        exception
+                           when others =>
+                              null;
```

Comment:
don't we want to forward the exception to the user though ? Silently dropping exception without even logging them seems bad ?